cmd/copy.go

```diff
@@ -1166,6 +1373,12 @@ func init() {
 	cpCmd.PersistentFlags().BoolVar(&raw.withSnapshots, "with-snapshots", false, "include the snapshots. Only valid when the source is blobs.")
 	cpCmd.PersistentFlags().StringVar(&raw.legacyInclude, "include", "", "only include these files when copying. "+
```

Comment:
TODO: rename the include/exclude flags to include-pattern and exclude-pattern, unhide the include one too